drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -409,6 +409,10 @@ msgid "ok"
 msgstr ""
 
 #: public/js/drag_and_drop.js
+msgid "Item Bank"
+msgstr ""
+
+#: public/js/drag_and_drop.js
 msgid "Placed in: "
```

Comment:
Shouldn't the placeholder be part of the string for translation?

Placed in: {}